### PR TITLE
configuration for SBR lease release #30090

### PR DESCRIPTION
**Pull request by patriknw**
References #30090

### CODE REVIEW

File: akka-cluster/src/main/resources/reference.conf [hunk 1]
```diff
@@ -495,6 +495,9 @@ akka.cluster.split-brain-resolver.lease-majority {
   # as an best effort to try to keep the majority side.
   acquire-lease-delay-for-minority = 2s
 
+  # Release the lease after this duration.
+  release-after = 40s
```
Comment (johanandren) [inline akka-cluster/src/main/resources/reference.conf:499]: Ok, so this _is_ default stable after * 2, but one thought: wasn't having it scaling with `stable-after` out of the box a good idea?

**Review by johanandren — APPROVED** [review]
LGTM with one question

**Review by octonato — APPROVED** [review]
LGTM

**Review by chbatey — APPROVED** [review]

File: akka-cluster/src/main/resources/reference.conf (hunk 1 above)
Comment (johanandren) [reply akka-cluster/src/main/resources/reference.conf:499]: Alright, lets go with a constant default like this 👍

**Comment by patriknw** [comment]
I had to change the structure how the property is accessed: 66747c7

**Pull request merged**